docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -240,7 +240,7 @@ This error occurs when an incorrect input port is specified with the `--input` c
 
 This error occurs when an incorrect combination of the `--input` and `--input_shape` command line options is used. Using both `--input` and `--input_shape` is valid only if `--input` points to the `Placeholder` node, a node with one input port or `--input` has the form `PORT:NODE`, where `PORT` is an integer port index of input for node `NODE`. Otherwise, the combination of `--input` and `--input_shape` is incorrect.
 
-#### 31. What does the message "Input port > 0 in --input is not supported if --input_shape is not provided. Node: NAME_OF_THE_NODE. Omit port index and all input ports will be replaced by placeholders. Or provide --input_shape" mean? <a name="question-31"></a>
+#### 31. (DEPRECATED)What does the message "Input port > 0 in --input is not supported if --input_shape is not provided. Node: NAME_OF_THE_NODE. Omit port index and all input ports will be replaced by placeholders. Or provide --input_shape" mean? <a name="question-31"></a>
```

Comment:
Let us do not write "DEPRECATED" here because this is not a feature and it has been a sort of limitation.

I would add in the end of this message: "It is no longer relevant message since the limitation on input port index for model truncation has been resolved."

Reply:
done